docs/source/module_configuration/staticsite.rst

```diff
@@ -43,71 +305,62 @@ Incorporating CloudFront with a static site is a common best practice, however,
 if you are working on a development project it may benefit you to add the
 `staticsite_cf_disable` environment parameter set to `true`.
 
-.. _staticsite-config-options:
+`Auth@Edge`
+-----------
 
-Example of all Static Site Options
-----------------------------------
+Authorization, the function of specifying access rights to resources, is often
+required to help protect restricted content in web applications. In a
+`blog post <https://aws.amazon.com/blogs/networking-and-content-delivery/authorizationedge-how-to-use-lambdaedge-and-json-web-tokens-to-enhance-web-application-security/>`_
+written by Alex Tomic and Cameron Worrell an in-depth solution is described
+on how to accomplish this with CloudFront, `Lambda@Edge` functions, and a
+Cognito User Pool. A similar solution has been integrated directly into
+Runway, via the StaticSite module, to allow easy and repeatable idempotent
+launching of private static sites.
```

Comment:
I'd prefer a slightly more concise overview here that just explains that the staticsite can be made private (authenticated by users in cognito or identity providers federated by cognito), and the solution is inspired by similar ones "for example: https://github.com/aws-samples/cloudfront-authorization-at-edge"

Reply:
Updated